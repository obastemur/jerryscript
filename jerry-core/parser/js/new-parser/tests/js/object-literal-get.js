/* This test contains object literal parser tests. */

var a = { get get(arg) {} }
