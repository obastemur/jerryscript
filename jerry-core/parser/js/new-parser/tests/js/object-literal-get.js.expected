
--- Script parsing start ---

  [  1] CBC_CREATE_OBJECT

--- Function parsing start ---


--- Script parsing end ---

Parse error 'Property getters must have no arguments.' at line: 3 col: 24
